front/app/components/ConsentManager/CookieModalContentContainer.tsx

```diff
@@ -9,7 +9,7 @@ interface Props {
   children: React.ReactNode;
 }
 
-const ContentContainer = ({ children, ...props }: Props) => {
+const CookieModalContentContainer = ({ children, ...props }: Props) => {
```

Comment:
It took me a while to understand why both modal screens had different styles. It turns out that one was wrapped with this `ContentContainer` while the other one was wrapped with `ContentContainer` from the `components` folder. I renamed this one to make the distinction clearer and the next person doesn't lose time.